app/src/main/res/values-fr/strings.xml

```diff
@@ -10,6 +10,7 @@
     <string name="vessel_name">Nom du vaisseau</string>
     <string name="vessel">Vaisseau</string>
     <string name="vessel_information">Détails du vaisseau</string>
+    <string name="add_an_attachment">Ajouter des pièces jointes</string>
```

Comment:
Change to:
"Ajouter une note ou une photo"

Reply:
Updated the string values.